app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -30,7 +30,6 @@ import mozilla.components.service.fxa.sync.SyncReason
 import mozilla.components.support.base.feature.UserInteractionHandler
```

Comment:
nit: a few notes on commit best practices:
- Commit messages should be manually wrapped around 80 characters; this is so they display correctly in a terminal. You can use the extended commit summary to provide more detailed information.
- When you make changes during a review as an additional commit, unless it's sufficiently complicated that the reviewer wants to see the diff (and sometimes they'll request this) or it'll take you too long to do it, it's often a good idea to squash them so there are no "fix nits" commits in the history (we don't usually squash commits when merging).
  - If the commit to change is on top, you can do `git commit --amend` to modify it
  - If the commit is somewhere in the middle, you have to do an interactive rebase: `git rebase -i <some-commit-above-the-commits-to-change>`; I usually do `git rebase -i HEAD~10` where 10 is some large number :)
- All of your commits should include the issue number (the way we merge is completely linear so you can't distinguish a PR without it